pom.xml

```diff
@@ -47,7 +47,7 @@
         <httpcVersion>4.4</httpcVersion>
         <atlasVersion>0.16.0</atlasVersion>
         <mockitoVersion>2.7.20</mockitoVersion>
-        <kerbyVersion>1.0.1</kerbyVersion>
+        <kerbyVersion>1.1.1</kerbyVersion>
```

Comment:
This fixes some bugs (so we can remove some local changes) and removes a transitive dependency (which we didn't use) that was insecure. I am trying to submit PRs upstream for the other changes, so we can eventually remove all local changes.